LanguageFeatures/Patterns/Exhaustiveness/lifting_constant_pattern_A02_t01.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion The lifted space union for a pattern with matched value type M is
+/// ...
+/// Constant pattern:
+/// i. If the constant has primitive equality, then a space whose type is the
+///   type of the constant and with a constant restriction for the given
+///   constant value.
+/// ii. Else the empty space union.
+///
+/// @description Check that if a constant have no primitive equality then its
+/// lifted space is an empty space
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=patterns
+
+class C {
+  final bool v;
+  const C(this.v);
+
+  bool operator ==(covariant bool other) => v == other;
+}
+
+const True = C(true);
```

Comment:
I think it's confusing that these constant variables have capitalized names. Perhaps `_true` and `_false` could be used, such that we don't violate basic expectations about capital letters?

Reply:
Renamed